### PR TITLE
WSTEAMA-478 - Launch: Simorgh - Route azeri to Homepage on live environment

### PR DESCRIPTION
**Pull request by LilyL0u**
Resolves JIRA [WSTEAMA-478]

Overall changes
======
Updates the logic to ensure that on all environments - local, test, live, that the homepage is rendered when service=azeri

Code changes
======

- Updated test so that /azeri renders a homepage not a frontpage
- Updated logic so that /azeri renders a homepage not a frontpage 
- Removed /azeri from cypress tests for frontpage on live environment by removing configuration and replacing with the 'empty' config as we won't need it for azeri anymore, but the object still needs to be present.

Testing
======
Test failed before Azeri was added to migratedServices in index.js, and now passes now that it is also added to migratedServices in the test.

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

File: src/app/routes/utils/regex/index.test.js [hunk 1]
```diff
@@ -550,7 +550,7 @@ describe('frontPage -> homePage migration', () => {
     .filter(service => !servicesWithVariants.includes(service))
     .map(serviceToRoute);
 
-  const migratedServices = ['kyrgyz'];
+  const migratedServices = ['kyrgyz', 'azeri'];
```
Comment (karinathomasbbc) [inline src/app/routes/utils/regex/index.test.js]: **nitpick**: Can we put these in alphabetical order?

File: src/app/routes/utils/regex/utils/index.js [hunk 2]
```diff
@@ -34,7 +34,7 @@ export const getArticleManifestRegex = services => {
   return `/:service(${serviceRegex})/:local(${articleLocalRegex})/manifest.json`;
 };
 
-const homePageServices = ['kyrgyz'];
+const homePageServices = ['kyrgyz', 'azeri'];
```
Comment (karinathomasbbc) [inline src/app/routes/utils/regex/utils/index.js]: As above - alphabetical order 🙏

**Review by karinathomasbbc — APPROVED** [review]
Wohoooo!!

**Review by louisearchibald — APPROVED** [review]

File: src/app/routes/utils/regex/index.test.js (hunk 1 above)
Comment (LilyL0u) [reply src/app/routes/utils/regex/index.test.js]: This is an excellent nitpick, good idea :)

File: src/app/routes/utils/regex/utils/index.js (hunk 2 above)
Comment (LilyL0u) [reply src/app/routes/utils/regex/utils/index.js]: done

**Pull request merged**